### ISSUE TITLE
flowtype/space-after-type-colon reports wrong errors

### ISSUE THREAD

**Issue by jpolo** [gajus/eslint-plugin-flowtype#85]
In version 2.11.1
```
interface RecordMethods<Spec: Object> {
  get<A>(key: $Keys<Spec>): A;
}
```
Reports an error 'There must be a space after "get" type annotation colon.
This errors only goes away if ```get< A>``` is set.

My eslint rules :
```
    'flowtype/define-flow-type': 'error',
    'flowtype/generic-spacing': ['error', 'never'],
    'flowtype/require-valid-file-annotation': ['error', 'always'],
    'flowtype/space-after-type-colon': ['error', 'always'],
    'flowtype/space-before-type-colon': ['error', 'never'],
    'flowtype/type-id-match': ['error', "^([\$]?[\A-Z][A-Za-z0-9]+)+$"],
```
